docs/tutorial/part-zero/index.md

```diff
@@ -67,43 +67,20 @@ The steps to download and install Git depend on your operating system. Follow th
 
 > 💡 You will not need to know Git to complete this tutorial, but it is a very useful tool. If you are interested in learning more about version control, Git, and GitHub, check out GitHub's [Git Handbook](https://guides.github.com/introduction/git-handbook/).
 
-## Install Gatsby CLI
+## Using the Gatsby CLI
 
-The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run commands for developing Gatsby sites. It is a published npm package. You can install Gatsby CLI from the npm registry, using the npm CLI.
+The Gatsby CLI tool lets you quickly create new Gatsby-powered sites and run commands for developing Gatsby sites. It is a published npm package. Rather than installing the Gatsby CLI tool locally you can run it using [npx](https://www.npmjs.com/package/npx)
 
-1.  Navigate to the terminal.
-2.  Run `npm install --global gatsby-cli`.
-
-> 💡 If you are unable to successfully install due to a permissions issue, you may want to check out the [npm docs on fixing permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions), or [this guide](https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md).
```

Comment:
Maybe keep this? But tweak the language, e.g.

> If you are unable to successfully use due to a permissions issue...

Reply:
Ok. Will it be confusing / unnecessary to explain how to install the CLI tool globally and verify the installation if we never use it?

All the other changes lead to a locally installed CLI never being used because it is always run via npx.

Reply:
Also thanks for your patience. A few sloppy bits of work in there. If you have any feedback on overall PR process definitely keen for your thoughts. First one for me so I'm muddling my way through a bit.

Reply:
True. I guess the same issues could still occur using npx.